packages/example-site/content/config/layout.json

```diff
@@ -38,6 +38,11 @@
         "name": "Search",
         "link": "/search/",
         "show": true
+      },
+      {
+        "name": "Newsletter",
+        "link": "https://mailchi.mp/12498740d69d/research-partnerships-portal",
+        "show": true
```

Comment:
Do we want the newsletter in the navbar, or just permanently pinned on the right?

Reply:
I think making it just a Navbar item seems the simplest. People can move it as they like by altering the listed order, and can (manually) move it to the right if desired. It's generalizable to having several external links too, if someone needs more. Does that answer the question?